main.js

```diff
@@ -1,30 +1,37 @@
-var sidesRecipes = ['carrot sticks', 'broccoli', 'peppers & onions', 'roti', 'dumplings'];
-var mainDishRecipes = ['vegan burger', 'grilled tofu', 'channa masala', 'tofu mutar masala'];
-var dessertRecipes = ['brownies', 'ice cream', 'Tonys Chocolonely', 'cookies', 'popsicles'];
-var favRecipes = ['pizza', 'muffins', 'ice cream'];
+var sidesRecipes = ['Carrot Sticks', 'Broccoli', 'Peppers & Onions', 'Roti', 'Dumplings', 'Mashed Potatoes', 'Corn on the Cob', 'Salad', 'Chips & Salsa', 'Black Beans & Rice', 'Smashed Potatoes', 'Baked Potatoes', 'Grilled Asparagus'];
+var mainDishRecipes = ['Pesto, Tomato & Mozzarella Veggie Burger', 'BBQ Grilled Tofu', 'Channa Masala', 'Tofu Mutar Masala', 'Tofu Fried Rice', 'Green Curry with Tofu', 'Red Curry with Tofu', 'Spicy Peanut Tofu', 'Crispy Tofu with Honey Mustard', 'Nachos with Black Beans & Guacamole'];
+var dessertRecipes = ['Apple Pie', 'Lemon Meringue Pie', 'Black Forest Cake', 'Banana Bread', 'Peach Cobbler', 'Cheesecake', 'Funfetti Cake', 'Baklava', 'Flan', 'Macarons', 'Macaroons', 'Chocolate Cupcakes', 'Pavlova', 'Pumpkin Pie', 'Key Lime Pie', 'Tart Tatin', 'Croissants', 'Eclairs'];
+var favRecipes = ['pizza', 'chocolate', 'sushi'];
 var currentRecipe = "";
 
 var viewFavsButton = document.querySelector('#view-favs-button');
 var letsCookButton = document.querySelector('#lets-cook-button');
 var letsCookBox = document.querySelector('#lets-cook-box');
 
-var sideOption = document.querySelector('#side-option').value;
-var mainDishOption = document.querySelector('#main-dish-option').value;
-var dessertOption = document.querySelector('#dessert-option').value;
-var entireMealOption = document.querySelector('#entire-meal-option').value;
+var sideOption = document.querySelector('#side-option');
+var mainDishOption = document.querySelector('#main-dish-option');
+var dessertOption = document.querySelector('#dessert-option');
+var entireMealOption = document.querySelector('#entire-meal-option');
 
 var cookpotImage = document.querySelector('.cookpot-image');
 var youShouldMake = document.querySelector('.you-should-make');
 var outputRecipe = document.querySelector('.output-recipe');
 var favsList = document.querySelector('.favs-list');
-var backButton = document.querySelector('.back-button');
 var favButton = document.querySelector('.fav-button');
+var deleteFavButton = document.querySelector('.delete-fav-button');
 var clearButton = document.querySelector('.clear-button');
+var backButton = document.querySelector('.back-button');
+
+var unorderedList = document.querySelector('.favs-list');
```

Comment:
Created new elements to show the favorited recipes in a list, with checkboxes available in case the user decides to delete them.